docs/contributor/03-11-hap-rules.md

```diff
@@ -0,0 +1,285 @@
+
+```
+hap: 
+  rule: 
+    - gcp -> S
+    - azure(PR=cf-ch20) -> E
+
+SecretBinding pools:
+- hyperscalerType: gcp; shared: true
+- hyperscalerType: azure_cf-ch20, euAccess: true
+```
+
+### Attribute with "*"
+
+Input attributes values can be replaced with `*` which means that all SKR values of that attribute are matched and reused in unchanged form when applying [search labels](#search-labels) modifications. For example, the following configuration makes KEB search SecretBindings with `hyperscalerType: gcp_<PR>`, where `PR` corresponds to an SKR value, for all `gcp` clusters:
+
+```
+hap: 
+  rule: 
+  - gcp(PR=*)
+```
+
+```
+SecretBinding pools:
+- hyperscalerType: gcp_cf-sa30
+- hyperscalerType: gcp_cf-jp30
+```
+
+The attributes that support `*` include: `PR`, `CR`.
+
+> ![NOTE]
+> The above configuration example effectively disables usage of SecretBindings with only `hyperscalerType: gcp` label.
+
+### Atributes Summary
+
+The following table summarizes the attributes that can be used in the rule entry. Its columns define:
+* Name and symbol - the later is used in rule entries.
+* Data type and possible values.
+* Input attributes - if value is true then the attribute can occure in the input attributes section (left part) of a rule entry (see [Rule Format](#rule-format) section).
+* Output attributes - if value is true then the attribute can occure in the output attributes section (right part) of a rule entry (see [Rule Format](#rule-format) section).
+* Modified SecretBinding search labels - lists modifications of search labels that the rule entry containing the attribute adds to search labels if triggered. 
+
+| Name (Symbol)        	| Data Type and Possible Values                                                                                                                    	| Input Attribute 	| Output Attribute 	| Modified SecretBinding Search Labels                                                              	|
+|----------------------	|------------------------------------------------------------------------------------------------------------------------------------	|-----------------	|------------------	|---------------------------------------------------------------------------------------------------	|
+| Platform Region (PR) 	| string, platform regions as defined in https://help.sap.com/docs/btp/sap-business-technology-platform/regions-for-kyma-environment 	| true            	| false            	| hyperscalerType: `<providerType>_<PR>` or hyperscalerType: `<providerType>_<PR>_<CR>` if used with CR 	|
+| Cluster Region (CR)  	| string, cluster region as defined in https://help.sap.com/docs/btp/sap-business-technology-platform/regions-for-kyma-environment   	| true            	| false            	| hyperscalerType: `<providerType>_<CR>` or hyperscalerType: `<providerType>_<PR>_<CR>` if used with PR 	|
```

Comment:
Change Cluster Region to  Hyperscaler Region. Cause in another parts of the document you use <HYPERSCALER_NAME> for plan name